migrations/2018_06_06_111024_create_chats_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateChatsTable extends Migration
```

Comment:
I'd recommend to add either class description or ```up()``` method description to DB migrations. Like 'Create table to store chats in system', etc.